Api/Product.php

```diff
@@ -20,8 +20,12 @@
 use Sulu\Bundle\CategoryBundle\Entity\Category as CategoryEntity;
 use Sulu\Bundle\MediaBundle\Api\Media;
 use Sulu\Bundle\ProductBundle\Entity\ProductAttribute as ProductAttributeEntity;
-use Sulu\Bundle\ProductBundle\Entity\ProductInterface as Entity;
 use Sulu\Bundle\ProductBundle\Entity\SpecialPrice as SpecialPriceEntity;
+use Sulu\Bundle\ProductBundle\Api\ProductAttribute;
```

Comment:
Please structure the use statements 

use Sulu\Bundle\ProductBundle\Api\*;
use Sulu\Bundle\ProductBundle\Api\*;
use Sulu\Bundle\ProductBundle\Api\*;
use Sulu\Bundle\ProductBundle\Api\*;
use Sulu\Bundle\ProductBundle\Entity\*;
use Sulu\Bundle\ProductBundle\Entity\*;
use Sulu\Bundle\ProductBundle\Entity\*;
use Sulu\Bundle\ProductBundle\Entity\*;